### PR TITLE
add french translation ^^

### PR DESCRIPTION
**Pull request by arthurbambou**

### CODE REVIEW

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "text.plasmid.game.started.player": "%s a démarré le jeu !",
+  "text.plasmid.game.stopped.player": "%s a arrêté le jeu !",
+  "text.plasmid.game.stopped.error": "Une erreur inattendue est survenu lors de l'arrêt du jeu !",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:4]:
```suggestion
  "text.plasmid.game.stopped.error": "Une erreur inattendue est survenue lors de l'arrêt du jeu !",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.game.list": "Jeux enregistrés :",
+  "text.plasmid.game.channel.create": "Canal '%s' créé",
+  "text.plasmid.game.channel.remove": "Canal '%s' supprimé",
+  "text.plasmid.game.channel.connect.entity": "'%1$s' connecté à '%2$s''",
+  "text.plasmid.game.channel.connect.block": "'%1$s' connecté au bloc avec pour position (%2$s; %3$s; %4$s)",
+  "text.plasmid.game.join": "%s a rejoint le lobby !",
+  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connection au jeu !",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:11]:
```suggestion
  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connexion au jeu !",
```

connection is English

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.game.open.error": "Une erreur inattendue est survenue lorsque que le jeu démarrait !",
+  "text.plasmid.game.open.opened": "%1$s a ouvert %2$s! ",
+  "text.plasmid.game.open.join": "Cliquez ici pour rejoindre",
+  "text.plasmid.game.waiting_lobby.bar.waiting": "En attente de joueurs...",
+  "text.plasmid.game.waiting_lobby.bar.countdown": "Démarre dans %s secondes !",
+  "text.plasmid.game.waiting_lobby.bar.cancel": "Démarrage annulé ! ",
+
+  "text.plasmid.game.start_result.already_started": "Ce jeu a déjà commencé !",
+  "text.plasmid.game.start_result.not_enough_players": "Il n'y a pas encore assez de joueurs pour démarrer !",
+  "text.plasmid.chat.switch.all": "Canal de discution réglé sur tous le monde",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:21]:
```suggestion
  "text.plasmid.chat.switch.all": "Canal de discussion règlé sur tous le monde",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.chat.switch.team": "Canal de discution réglé sur l'équipe",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:22]:
```suggestion
  "text.plasmid.chat.switch.team": "Canal de discussion règlé sur l'équipe",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+  "text.plasmid.party.error.does_not_exist": "Vous n'avez le contrôle d'aucune partie !",
+  "text.plasmid.party.error.already_invited": "%s est déjà invité(e) dans votre partie !",
+  "text.plasmid.party.error.already_in_party": "Vous faites déjà parti de cette partie !",
+  "text.plasmid.party.error.cannot_remove_self": "Vous ne pouvez pas vous enlever de votre propre partie !",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:27]:
```suggestion
  "text.plasmid.party.error.cannot_remove_self": "Vous ne pouvez pas quitter votre propre partie !",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.party.error.not_in_party": "%s ne fait pas parti de cette partie !",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:28]:
```suggestion
  "text.plasmid.party.error.not_in_party": "%s ne fait pas parti(e) de cette partie !",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.party.error.not_invited": "Vous n'êtes pas invité(e) à cette partie !",
+
+  "text.plasmid.party.invited.sender": "%s a été invité(e) dans la partie",
+  "text.plasmid.party.invited.receiver": "Vous avez été invité(e) à rejoindre la partie de %s' ! ",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:32]:
```suggestion
  "text.plasmid.party.invited.receiver": "Vous avez été invité(e) à rejoindre la partie de %s !",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.party.invited.receiver.click": "Cliquez ici pour la rejoindre",
+  "text.plasmid.party.invited.receiver.hover": "Rejoindre la partie de %s !",
+
+  "text.plasmid.party.kicked.sender": "%s a été éjecté(e) de la partie",
+  "text.plasmid.party.kicked.receiver": "Vous avez été éjecté(e) de la partie"
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:37]:
```suggestion
  "text.plasmid.party.kicked.receiver": "Vous avez été éjecté de la partie."
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.game.channel.connect.entity": "'%1$s' connecté à '%2$s''",
+  "text.plasmid.game.channel.connect.block": "'%1$s' connecté au bloc avec pour position (%2$s; %3$s; %4$s)",
+  "text.plasmid.game.join": "%s a rejoint le lobby !",
+  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connection au jeu !",
+  "text.plasmid.game.open.error": "Une erreur inattendue est survenue lorsque que le jeu démarrait !",
+  "text.plasmid.game.open.opened": "%1$s a ouvert %2$s! ",
+  "text.plasmid.game.open.join": "Cliquez ici pour rejoindre",
+  "text.plasmid.game.waiting_lobby.bar.waiting": "En attente de joueurs...",
+  "text.plasmid.game.waiting_lobby.bar.countdown": "Démarre dans %s secondes !",
+  "text.plasmid.game.waiting_lobby.bar.cancel": "Démarrage annulé ! ",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:17]:
```suggestion
  "text.plasmid.game.waiting_lobby.bar.cancel": "Démarrage annulé !",
```

File: src/main/resources/data/plasmid/lang/fr_fr.json [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+  "text.plasmid.game.stopped.error": "Une erreur inattendue est survenu lors de l'arrêt du jeu !",
+  "text.plasmid.game.list": "Jeux enregistrés :",
+  "text.plasmid.game.channel.create": "Canal '%s' créé",
+  "text.plasmid.game.channel.remove": "Canal '%s' supprimé",
+  "text.plasmid.game.channel.connect.entity": "'%1$s' connecté à '%2$s''",
+  "text.plasmid.game.channel.connect.block": "'%1$s' connecté au bloc avec pour position (%2$s; %3$s; %4$s)",
+  "text.plasmid.game.join": "%s a rejoint le lobby !",
+  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connection au jeu !",
+  "text.plasmid.game.open.error": "Une erreur inattendue est survenue lorsque que le jeu démarrait !",
+  "text.plasmid.game.open.opened": "%1$s a ouvert %2$s! ",
```
Comment (LambdAurora) [inline src/main/resources/data/plasmid/lang/fr_fr.json:13]:
```suggestion
  "text.plasmid.game.open.opened": "%1$s a ouvert %2$s!",
```

**Review by LambdAurora — CHANGES_REQUESTED** [review]
To be honest, I really don't like spaces before `!` or `:` but it's French convention so it's ok.

File: src/main/resources/data/plasmid/lang/fr_fr.json (hunk 8 above)
Comment (arthurbambou) [reply src/main/resources/data/plasmid/lang/fr_fr.json:37]: why removing the "(e)"?

File: src/main/resources/data/plasmid/lang/fr_fr.json (hunk 9 above)
Comment (arthurbambou) [reply src/main/resources/data/plasmid/lang/fr_fr.json:17]: this space is needed, it's in the english translation too.

File: src/main/resources/data/plasmid/lang/fr_fr.json (hunk 2 above)
Comment (arthurbambou) [reply src/main/resources/data/plasmid/lang/fr_fr.json]: ~~how do I always miss this one~~

File: src/main/resources/data/plasmid/lang/fr_fr.json (hunk 8 above)
Comment (LambdAurora) [reply src/main/resources/data/plasmid/lang/fr_fr.json]: My bad, misread.

**Review by LambdAurora — APPROVED** [review]

**Review by gegy1000 — APPROVED** [review]

**Pull request merged**